src/SFA.DAS.AssessorService.Application/Interfaces/IEpaOrganisationSearchValidator.cs

```diff
@@ -0,0 +1,9 @@
+﻿namespace SFA.DAS.AssessorService.Application.Interfaces
+{
+    public interface IEpaOrganisationSearchValidator
+    {
+        bool IsValidEpaOrganisationId(string organisationIdToCheck);
+
```

Comment:
Not sure vertical space between methods adds clarity

Reply:
No it doesnt :D